picturae_csv_create.py

```diff
@@ -434,19 +447,51 @@ def taxon_check_tnrs(self):
         if self.records_dropped > 0:
             self.logger.info(f"{self.records_dropped} rows dropped due to taxon errors")
 
+        self.cleanup_tnrs()
+
```

Comment:
split off cleanup code for tnrs, which reconsolidates columns post-TNRS and updates taxon-id + outdated infraspecific ranks.